includes/html/modal/new_bill.inc.php

```diff
@@ -44,12 +44,18 @@
                         <select class="form-control input-sm" id="device" name="device" onchange="getInterfaceList(this)">
                             <option value=''>Select a device</option>
                             <?php
-                              $devices = dbFetchRows('SELECT * FROM `devices` ORDER BY hostname');
-                            foreach ($devices as $device) {
-                                $selected = $device['device_id'] == $port_device_id ? " selected" : "";
-                                echo "<option value='${device['device_id']}' $selected>${device['hostname']}</option>\n";
-                            }
-                                ?>
+                              $sort_field = 'hostname';
+
+                              if (\LibreNMS\Config::get('force_ip_to_sysname') === true) {
+                                  $display_field = 'sysName';
+                              }
+
+                              $devices = dbFetchRows('SELECT * FROM `devices` ORDER BY ?', array($display_field));
```

Comment:
Hi @TvL2386 
Thanx for your PR ! It seems that there are still some variable to rename. 
:)

Reply:
Thanks PipoCanaja, I feel ashamed...